test/integration/ae_mdw_web/controllers/activity_controller_test.exs

```diff
@@ -183,4 +183,26 @@ defmodule Integration.AeMdwWeb.ActivityControllerTest do
              ] = events
     end
   end
+
+  test "it gets tx-based internal transfer events", %{conn: conn} do
```

Comment:
Could we have an extra one that would have both generational and transactional internal transfer events and test the pagination if it skips both internal transfers on the next page?

Reply:
I searched the entire transfers database and wasn't able to find a generation that contains both internal and external transfers from the same account

Reply:
Thanks, could you add in the CI then? It's possible for an account that receives a dev reward to also create an oracle, right?

Reply:
Added a unit tests with both cases combined now, thanks